docs/standalone.md

```diff
@@ -1,3 +1,240 @@
-## Running Heapster standalone on a machine
+Runnig Heapster standalone on a host with cAdvisor
+================================
+
+- [Introduction](#introduction)
+- [Prerequisites](#prerequisites)
+    - [Prepare Heapster binaries](#prepare-heapster-binaries)
+    - [Running a Cadvisor](#running-a-cadvisor)
+- [Starting a heapster](#starting-a-heapster)
+    - [Config Cadvisor source](#config-cadvisor-source)
+    - [Start heapster](#start-heapster)
+    - [Test it out](#test-it-out)
+
+## Introduction
+
+This document describes how to to run Heapster standalone with [cAdvisor](https://github.com/google/cadvisor). These guide is tested OK on Ubuntu 14.04 LTS 64bit server. I'm working around testing this with some other system . The assumptions here are that [influxdb](https://github.com/influxdb/influxdb), and [grafana](https://github.com/grafana/grafana.git) run on one machine, which heapster can visit.
+
+## Prerequisites
+
+### Prepare Heapster binaries
+We can get the heapster binary from the [Release Page](https://github.com/kubernetes/heapster/releases/) or build the heapster from source code.The build steps are as follows, [go](https://github.com/golang) and [godep](https://github.com/tools/godep) need be installed in system first:
+
+
+**Step 1: Clone the heapster github repo**
+
+``` 
+$ git clone https://github.com/kubernetes/heapster.git
+```
+
+**Step 2: Build the source code**
+
+``` 
+$ cd kubernetes/heapster
+$ godep go build
```

Comment:
Should rather run `make` instead of `godep go build` - right now it is more or less equivalent but this may possibly change in the future.